docs/blog/newsletter-july-2019.rst

```diff
@@ -0,0 +1,115 @@
+
+A few other people also weighed in with some comments. Some don't work with a 1099 but set up their own corporations (S-Corp or LLC), and then work as their own employees. This approach can provide tax benefits for US workers, and can sometimes make you more appealing to a larger employer. People also suggested that professional support, in the form of a lawyer or accountant, is important if you're planning to work freelance long-term.
+
+And if you're going the independent route, don't forget to consider staffing agencies. They get a cut of whatever the client pays, but they sometimes provide benefits depending on the contract, and can ease the paperwork burden.
+
+------------------------------
+What to do about text wrapping
+------------------------------
+
+A very specific topic, but one that comes up for almost everyone working with docs-as-code: should you mandate text wrapping in your docs source, and if so, how? For some background before digging into this summary, check out `To Wrap or Not to Wrap: One of Life's Greatest Questions <https://about.gitlab.com/2016/10/11/wrapping-text/>`_ 
+
+Forcing a hard wrap at a particular character count or column helps avoid `reflow problems <https://www.w3.org/TR/WCAG21/#reflow>`_, but causes other problems. For example, hard wraps:
+
+* create awkward line breaks on devices with different screen sizes
+* wreak havoc on diffs
+* make text more difficult to parse for people who are translating to different languages and people with cognitive disabilities
+
+Hard wraps also interfere with using filters like `grep <http://man7.org/linux/man-pages/man1/grep.1.html>`_ to search for words and phrases within a file. Soft wrap is an option for preserving searchability, but it requires everyone to use a text editor with soft wrap support.
+
+Several people mentioned `one sentence per line <https://rhodesmill.org/brandon/2012/one-sentence-per-line/>`_ as a solution, noting that it's the `recommended approach for AsciiDoc <https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/asciidoc-recommended-practices.adoc#one-sentence-per-line>`_.
+
+-----------------------------------------------------------------
+From our sponsor: Spotlight on The Not-Boring Tech Writer podcast 
+-----------------------------------------------------------------
+
+Excitingly, this month's newsletter is sponsored by KnowledgeOwl!
+
+"Do you love documentation and podcasts? Check out our `spotlight on The Not-Boring Tech Writer podcast <https://www.knowledgeowl.com/home/not-boring-tech-writer-podcast-relaunch>`__. Each episode focuses on a single skill or tool to provide you with actionable content.
+
+KnowledgeOwl is proud to sponsor TNBT podcast along with the WTD newsletter. KnowledgeOwl makes knowledge base software and loves to help documentarians. Check out the `KnowledgeOwl website <https://www.knowledgeowl.com/home>`__ to learn more."
+
+.. image:: knowledge-owl-logo.png
```

Comment:
@ericholscher, can you take a look at ways to make this look nice and not just be a big image dumped in the middle?

Reply:
Yep, I've pushed an update. Email is terrible, so it's pretty hacky, but not too bad.

Reply:
You can throw html in using

```
.. raw:: html
   <imr src.... >
````
if that helps?